tzupdate.py

```diff
@@ -56,6 +56,47 @@ def get_deep(item, keys):
     return tmp
 
 
+def get_timezone(ip, timeout=5.0, services=SERVICES):
+    """
```

Comment:
No need for this big docstring, since we're not really supposed to be called as a library :-) You can just remove it all, it's fairly self-explanatory.